### ISSUE TITLE
YAMLException: unknown escape sequence on regex value

### ISSUE THREAD

**Issue by whallify** [alexlafroscia/yaml-merge#8]
Sample file contains array of regexes to be used.

```
logger:
    myfilter:    [ "^/api/device$", "^/sys/log(in|out)", "/api/thing/\w+/device/\w+" ]
```

Throws error:

```
YAMLException: unknown escape sequence at line xx, column yy:
     ... sys/log(in|out)", "/api/thing/\w+/device/\w+" ]
                                        ^
```

Commenting out line is a workaround, since it seems the tool removes all comments

**Comment by alexlafroscia** [comment]
I just released a new version (`2.0.0`) that in part bumps our internal `js-yaml` dependency to the latest version.

That tool does all the parsing of YAML files and all, so if that doesn't solve your problem, I would open an issue on that package. There's very little YAML-specific logic in here!

**Comment by moqmar** [comment]
Seems to me like that file is actually invalid: you will have to escape the backslash (`\\w`) so that the parsed string corresponds to `\w`.

**Comment by alexlafroscia** [comment]
Indeed; using http://www.yamllint.com says that the input YAML is invalid.
